index.html

```diff
@@ -180,7 +180,7 @@ <h2 class="card-header">About me</h2>
       <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
     </header>
-    <form id="contact-form" action="" target="_self" method="post">
+    <form id="contact-form" action="https://formspree.io/f/meqwrjep" target="_self" method="post">
       <fieldset>
           <input id="forminput1" type="text" name="full_name" placeholder="Full Name" required maxlength="30" />
           <label for="forminput1" class="label-remove">Full Name</label>
```

Comment:
- [ ] While you make a great effort 😃 I see that the margin between the submit button and the input is greater than the Figma design .so please try to decrease the margin between the input and the submit button to make it like the Figma Design 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/228208846-347930ca-4436-4ac3-b6b3-2be9918b9522.png)    |    ![image](https://user-images.githubusercontent.com/83360636/228208793-2a85e9da-1410-44db-bfd3-eddea7887c08.png) |

Reply:
sir this version is only optimized for the mobile